### PR TITLE
GH4252: Update .NET SDK to 8.0.100

### LINKED ISSUE #4252: Update .NET SDK to 8.0.100

### PR DESCRIPTION
**Pull request by devlead**
* fixes #4252

Known issues so far
- [x] .NET 8 Removes support for CentOS 7

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -25,6 +25,11 @@ jobs:
         with:
           dotnet-version: '6.0.x'
 
+      - name: Install .NET SDK 7.0.x
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '7.0.x'
+
```
Comment (nils-a) [inline .github/workflows/build.yml:32]:
```suggestion
        with:
          dotnet-version: |
            '6.0.x'
            '7.0.x'
```

**Review by nils-a — APPROVED** [review]
LGTM!

File: .github/workflows/build.yml (hunk 1 above)
Comment (devlead) [reply .github/workflows/build.yml]: adjusted, needs to be without quotes for multiple versions.

File: src/Cake.Frosting.Template/templates/cakefrosting/.template.config/template.json [hunk 2]
```diff
@@ -27,6 +27,10 @@
             {
             "choice": "net7.0",
             "description": "Target .NET 7"
+            },
+            {
+            "choice": "net8.0",
+            "description": "Target .NET 7"
```
Comment (gep13) [inline src/Cake.Frosting.Template/templates/cakefrosting/.template.config/template.json:33]:
```suggestion
            "description": "Target .NET 8"
```

**Review by gep13 — CHANGES_REQUESTED** [review]

File: src/Cake.Frosting.Template/templates/cakefrosting/.template.config/template.json (hunk 2 above)
Comment (devlead) [reply src/Cake.Frosting.Template/templates/cakefrosting/.template.config/template.json]: Fixed.

**Review by gep13 — COMMENTED** [review]
LGTM!

**Pull request merged**